### DISCUSSION TITLE
Fields: abstract / description and new field.

### DISCUSSION THREAD

**Discussion by meteos77** [textbrowser/biblioteq#162] [Ideas]
concerning this field (abstract /description) I see rather the abstract information (field 330$a)
example : 
"Epopée lyrique et saga familiale, ce roman constitue le premier volet d'une trilogie consacrée à Homewood, quartier noir de Pittsburg où l'auteur a passé sa jeunesse. Il retrace le destin d'une dynastie ayant ses racines dans ce quartier où s'installa, vers 1840, un couple mixte formé d'une esclave évadée et du fils de son maître"

on the other hand the fields of physical quality of the document
unimarc 215$a : Type of physical presentation  ( Volume and Pages ;  example : 1 vol. (535 p.)
unimarc 215$d : Dimensions ; example : 25 cm

there are no fields available

**Comment by meteos77** [comment]
I even thought of a new trick to contain all these practical but rather secondary fields

!! "I don't know if it's a mountain or a hill :-) !

the idea came with the "enumeration database" that fills itself and the search function that proposes directly the fields added in the "location" field

suggestion : 
add a widget of type: Table widget in the book entry window

so a table with 3 columns : id ((automatic 1 2 3 ... ) ; field name ; field data

these data will be present only in this view (book entries); in the import / export and in the sql request.
- advantages : no new fields at each small unimarc field
- user can add what he wants in his small table
- the exploitation is sufficient (with import / export / sql)
-
in pictures
![suggestion-162](https://user-images.githubusercontent.com/56431039/147744652-012388f2-6a74-4d14-bb98-36f13006eed1.png)

**Comment by textbrowser** [comment]
Indeed, quite a lot of work.

**Comment by meteos77** [comment]
I don't want you to run out of ideas :-)

when I see the work that needs to be done to add a field like "target audience". I think we will go through you long enough to manage our other needs

**Comment by meteos77** [comment]
A ticket has been opened for this feature : New fields, a request. #260 

My project for this example,

![BUT](https://github.com/textbrowser/biblioteq/assets/56431039/3817c075-4f2e-43df-ac75-c5df6e974edb)

create a newletter with the news of our library.

I was not able to extract the images from the SQLite database : This will be a project for 2024
